### PR TITLE
Update selectors

### PR DESCRIPTION
**Pull request by lufernando100**

### CODE REVIEW

File: cypress/page/index.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export { MenuContentPage } from "./menu-content-page";
+export {AddressStepPage } from "./address-step-page";
+export {PaymentStepPage } from "./payment-step-page";
+export {ProductsListPage } from "./products-list-page";
+export {ShippingStepPage } from "./shipping-step-page";
+export {ShoppingCartPage } from "./shoping-cart-page";
+
+
```
Comment (manuelq12) [inline cypress/page/index.ts]: Please erase this blank lines

File: cypress/page/products-list-page.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  selectProductList(): void {
+    cy.get(this.productList).click();
+  }
+  goToShopping(): void{
+    cy.get(this.proceedToCheckoutButton).click();
+  }
+}
+ 
+export{ProductsListPage}
+
```
Comment (manuelq12) [inline cypress/page/products-list-page.ts:19]: This blank line too

File: cypress/page/menu-content-page.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+class MenuContentPage {
+  private tShirtMenu: string;
+  private menuContentPageURL: string;
+
+  constructor() {
+    this.tShirtMenu = "li:nth-child(3) [title='T-shirts']";
+    this.menuContentPageURL = "http://automationpractice.com/";
+    cy.pause
```
Comment (manuelq12) [inline cypress/page/menu-content-page.ts]: Why is there a cy.pause here?

File: cypress/page/menu-content-page.ts [hunk 4]
```diff
@@ -0,0 +1,20 @@
+class MenuContentPage {
+  private tShirtMenu: string;
+  private menuContentPageURL: string;
+
+  constructor() {
+    this.tShirtMenu = "li:nth-child(3) [title='T-shirts']";
```
Comment (manuelq12) [inline cypress/page/menu-content-page.ts:6]: The nth-child is not necessary as far as I remember

File: cypress/page/payment-step-page.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+class PaymentStepPage {
+  private payByBankButton: string;
+  private confirmOrderButton: string;
+  private confirmationMessage: string;
+
+  constructor() {
+    this.payByBankButton = ".bankwire";
+    this.confirmOrderButton = "#cart_navigation .button-medium";
+    this.confirmationMessage = ".cheque-indent > .dark";
```
Comment (manuelq12) [inline cypress/page/payment-step-page.ts:9]: Do you require the .dark?

File: cypress/page/shipping-step-page.ts [hunk 6]
```diff
@@ -0,0 +1,16 @@
+class ShippingStepPage {
+  private agreeTermsCheckbox: string;
+  private proceedCheckoutButton: string
+
+  constructor() {
+    this.agreeTermsCheckbox = "#cgv";
+    this.proceedCheckoutButton = ".cart_navigation > .button > span";
```
Comment (manuelq12) [inline cypress/page/shipping-step-page.ts]: I think span can be erased

File: cypress/page/shoping-cart-page.ts [hunk 7]
```diff
@@ -0,0 +1,23 @@
+class ShoppingCartPage {
+  private checkoutButton: string;
+  private useraname: string;
+  private password: string;
+  private signInButton: string;
+
+  constructor() {
+    this.checkoutButton = ".standard-checkout > span";
```
Comment (manuelq12) [inline cypress/page/shoping-cart-page.ts]: I think span can be erased

**Review by manuelq12 — CHANGES_REQUESTED** [review]

File: cypress/page/payment-step-page.ts (hunk 5 above)
Comment (lufernando100) [reply cypress/page/payment-step-page.ts:9]: Yes, test fails if I deleted .dark

File: cypress/page/menu-content-page.ts (hunk 4 above)
Comment (lufernando100) [reply cypress/page/menu-content-page.ts:6]: test fails if I remove the nth-child

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**